api/zulip/__init__.py

```diff
@@ -54,34 +53,40 @@
 API_VERSTRING = "v1/"
 
 class CountingBackoff(object):
-    def __init__(self, maximum_retries=10, timeout_success_equivalent=None):
-        self.number_of_retries = 0
+    def __init__(self, maximum_retries=10.0, timeout_success_equivalent=None):
+        # type: (float, float) -> None
+        self.number_of_retries = 0.0
```

Comment:
`number_of_retries` and the related values should be tagged as an int, not a float, since that's what it is.